### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Alexandre Philibert, Gwendal Piemontesi, Valentin Ricard, Trueb Guillaume

### DISCUSSION THREAD

**Discussion by tasty-orange** [heig-vd-dai-course/heig-vd-dai-course#182] [Show and tell]
We will make an ice cream truck simulator.

**Comment by AlexandrePhilibert** [comment]
Link to the repo : https://github.com/AlexandrePhilibert/dai-labo-4/
Last commit [dd632f2c385b8d21d142a11edd9f0879bc65c41a](https://github.com/AlexandrePhilibert/dai-labo-4/commit/dd632f2c385b8d21d142a11edd9f0879bc65c41a)
